libs/components/progress-indicator/src/lib/modules/progress-indicator/progress-indicator-status-marker/progress-indicator-status-marker.component.scss

```diff
@@ -7,6 +7,9 @@
 $sky-progress-indicator-success-color: #5a9936;
 
 .sky-progress-indicator-status-marker {
+  --sky-progress-indicator-marker-line-icon-color: var(
+    --sky-border-color-neutral-medium
```

Comment:
Is there a reason we are leaving this for default instead of it also being `medium-dark`? We didn't on the other two stories and I don't see that called out in the story.

Reply:
I understand now - disregard